pupil_src/player/main.py

```diff
@@ -149,13 +151,20 @@ def session(rec_dir):
                               Pupil_Angle_3D_Fixation_Detector,
                               Manual_Gaze_Correction, Video_Export_Launcher,
                               Offline_Surface_Tracker, Raw_Data_Exporter,
-                              Batch_Exporter, Annotation_Player,
-                              Offline_Pupil_Detection, Offline_Calibration], key=lambda x: x.__name__)
+                              Batch_Exporter, Annotation_Player], key=lambda x: x.__name__)
+
+    global pupil_producers
+    global gaze_producers
+
+    pupil_producers += [p for p in runtime_plugins if issubclass(p, Pupil_Producer_Base)]
```

Comment:
I d rather not have these lists here. lets refactor the plugin logic so that spliting into categories happens via a base class in the menu selector and not explicitly in code here.